_content/monographs/seon-dialogues_jorgensen-john.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Seon Dialogues 禪語錄"
+translator: "John Jorgensen"
+external_url: "https://siteofenlightenment.org/Tripitaka/Books/More/Collected%20Works%20of%20Korean%20Buddhism/08_Seon%20Dialogues_web.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/18a4lSNo_hIAabaqZDS65qkZa1jGiLwW3/view?usp=sharing"
+course: korean
+tags:
+  - chan-lit
+year: 2012
+month: jun
+olid: OL60557309M
+publisher: "Jogye Order of Korean Buddism" 
```

Comment:
Yes, monographs with this publisher is okay. Yes please add the series and number: 8